airflow/www/templates/airflow/trigger.html

```diff
@@ -27,11 +27,93 @@
   <link rel="stylesheet" type="text/css" href="{{ url_for_asset('switch.css') }}">
 {% endblock %}
 
+{% macro form_element(form_key, form_details) %}
+<tr>
+  <td class="col-lg-2">
+    <label for="element_{{ form_key }}" control-label="">
+      {%- if "title" in form_details.schema and form_details.schema.title -%}
+        {{ form_details.schema.title }}
+      {%- else -%}
+        {{ form_key }}
+      {%- endif -%}
+      {%- if form_details.schema and form_details.schema.type and not "null" in form_details.schema.type and not "boolean" in form_details.schema.type -%}
+        <strong style="color: red"> *</strong>
+      {%- endif -%}
+    : </label>
+  </td>
+  <td>
+    {% if "custom_html_form" in form_details.schema %}
+      {{ form_details.schema.custom_html_form | replace("{name}", "element_" + form_key) | replace("{value}", form_details.value) }}
+    {% elif "type" in form_details.schema and form_details.schema.type == "boolean" %}
+      <label for="element_{{ form_key }}" control-label="">
+        <input class="switch-input" name="element_{{ form_key }}" id="element_{{ form_key }}" type="checkbox"
+          {%- if form_details.value %} checked="checked"{%- endif -%}/>
+        <span class="switch" aria-hidden="true"></span>
+      </label>
+    {% elif "format" in form_details.schema and "date-time" in form_details.schema.format %}
+      <div class="input-group datetime datetimepicker">
+        <span class="input-group-addon"><span class="material-icons cursor-hand">calendar_today</span></span>
+        <input class="form-control" name="element_{{ form_key }}" id="element_{{ form_key }}" type="text" valuetype="date"
+          {%- if not "null" in form_details.schema.type %} required=""{%- endif %} value="{% if form_details.value %}{{ form_details.value }}{% endif %}" />
+      </div>
```

Comment:
These lines render a date-time picker (Unfortunately I'm not able to screen-capture the pupp of the date/time picker but it is the same like on other UI parts:
![image](https://user-images.githubusercontent.com/95105677/214950315-971300b7-9448-4630-a83d-7a359ae7346b.png)